lib/taurus/qt/qtcore/util/emitter.py

```diff
@@ -23,10 +23,20 @@
 ###########################################################################
 
 """
-emitter.py: This module provides a task scheduler used by TaurusGrid and TaurusDevTree widgets
+emitter.py: This module provides a task scheduler used by TaurusGrid and 
+    TaurusDevTree widgets
 """
 
-import Queue
+try:
+    import queue
```

Comment:
Please use only 1 flavour (I suggest the modern one: `queue`) .
According to https://docs.python.org/2/library/queue.html , the `Queue` module was renamed to `queue` **in python 3**